spec/spec_helper.rb

```diff
@@ -18,4 +18,5 @@
 
 RSpec.configure do |config|
   config.warnings = true
+  config.filter_run_when_matching :focus
```

Comment:
FYI this require rspec core >= 3.5

https://relishapp.com/rspec/rspec-core/v/3-5/docs/filtering/filter-run-when-matching (switch version to 3.4 and docs can't be found)

the Gemfile here says rspec 3.0: https://github.com/sporkmonger/addressable/blob/b919fb0ad73a0eb9add6c0d3f07a6f14a414ecf5/Gemfile#L6

I think that should be bumped if this goes in

Reply:
@dentarg  you know better what is better for this repo
Tell me to change the version of the gem or delete this line?

I will do as you say, thank you